### ISSUE TITLE
BIGIP ASM LOGS 

### ISSUE THREAD

**Issue by khulood94** [f5devcentral/f5-bados-app#2]
Hi Alexander, 

I'm running bigip v 13 with evergreen , all virtual servers details are showing except asm logs includes attack signature details.
is there any further steps should be done in bigip server to track attack signatures logs ? 

note that , i did nothing related to Dev install and App creation as you mentioned in the post.Are these steps necessary? and for what it used?

Thanks in advance.

**Comment by akruman** [comment]
Hi, f5 does not officially support this grafana application.
As i remember evergreen does not have the attack signatures feature.
dev is just for me :)

**Comment by khulood94** [comment]
Hi Akruman, 
But there is build in dashboard for attack signatures , the query is already there but no data is showing.  
even for DDOS no data is showing. Only virtual server list and system logs are showing.

**Comment by akruman** [comment]
Hi, did you do the steps below ? (they are mentioned somewhere in ui of grafana, in home screen of the app, or in the datasource):
To add new BIG-IP to for visualization please follow these steps:   
<li>SSH to your BIG-IP</li>    
<li>tmsh modify sys db adm.cloud.host value local</li>    
<li><a href=\"/datasources/new\">Add new data source</a></li></ul></div>To enable traffic samples run the following command on BIG-IP:<br/><br/><code>tmsh modify sys db admdb.debug.traffic.sample value enable</code>

**Comment by khulood94** [comment]
yes I have , but only virtual server and pool details are showing . But I'm not able to show security logs.
